src/components/nodes/sequenceFlow/sequenceFlow.vue

```diff
@@ -79,15 +94,17 @@ export default {
   watch: {
     'node.definition': {
       handler({ startEvent: startEventId, conditionExpression }) {
-        if (!this.sourceIsGateway && !this.targetIsCallActivity) {
+        if (this.targetIsCallActivity) {
           return;
         }
 
         const startEvent = store.getters.globalProcessEvents.find(event => event.id == startEventId);
-        const newLabel = get(conditionExpression, 'body') || get(startEvent, 'name');
+        const newExpressionLabel = get(conditionExpression, 'body') || get(startEvent, 'name');
```

Comment:
Wonder if we can drop the `get(startEvent, 'name')`. In what scenario would this appear?

Reply:
Connecting a start-event to a sub-process will trigger this scenario.